utils/interfaces/_library/appsec_iast.py

```diff
@@ -19,6 +19,7 @@ class _BaseAppSecIastValidation(BaseValidation):
     def __init__(self, request):
         super().__init__(request=request)
         self.appsec_iast_events = []  # list of all appsec iast events
+        self.request = request
```

Comment:
I called request like in another python files, but, indeed, wouldn't it be a response?

Reply:
You are perfectly right ...

Though, it'll be for another PR, pretty big one indeed ...